pkg/web/transactions.go

```diff
@@ -423,3 +423,41 @@ func CheckUUIDMatch(id, target uuid.UUID) error {
 
 	return nil
 }
+
+func (s *Server) TransactionInfo(c *gin.Context) {
+	var (
+		err           error
+		transactionID uuid.UUID
+		transaction   *models.Transaction
+		out           *api.Transaction
+	)
+
+	// Parse the transactionID passed in from the URL
+	if transactionID, err = uuid.Parse(c.Param("id")); err != nil {
+		c.JSON(http.StatusNotFound, api.Error("transaction not found"))
```

Comment:
Should this be a 400 instead of a 404 because the ID was not a valid UUID? I could probably be convinced either way, so I guess it depends on how the other endpoints are handling it.

Reply:
A 404 has been returned in other endpoints, so that's why it's used here.